packages/components/src/components/attribution/overlay.tsx

```diff
@@ -35,7 +43,7 @@ const AttributionOverlay = ({ gif, isHovered }: { gif: IGif; isHovered: boolean
     return gif.user && hasHovered.current ? (
         <div className={containerCss} style={{ opacity: isHovered ? 1 : 0 }}>
             <div className={backgroundCss} />
-            <Attribution gif={gif} className={attributionCss} />
+            <Attribution gif={gif} className={attributionCss} onClick={onClick ? onClick(gif) : undefined} />
```

Comment:
@nicksantan shouldn't this just be `onClick={onClick}`

Reply:
Looks like I tried to get too complicated. Change has been made -